templates/acronyms.haml

```diff
@@ -1,20 +1,21 @@
 %h2#top Acronyms &amp; Abbreviations
 
-%p
+%h3
```

Comment:
I don't think `<h3>` is right here, since semantically this means there is a section titled "A B C D E F G H I J K L M N O P Q S T U V W". It might be better creating a new class like `.big` to apply the larger size. If we were being strict about it then it should belong in a `<nav>` element, but that seems like part of a further round of improvements.

Reply:
Thanks! I went with the class `.h3`, to show that this is how the thinking started - we can change it to something more semantic later.